caom2repo/tests/test-integration.py

```diff
@@ -76,26 +76,26 @@
 import binascii
 import os
 import sys
+import logging
 
 from cadcutils.net import auth
-from cadcutils.util import utils
-from caom2repo import CAOM2RepoClient
-from io import BytesIO
-from caom2 import common
-from caom2 import chunk
-from caom2 import part
-from caom2 import plane
-from caom2 import artifact
+from caom2repo.core import CAOM2RepoClient
 from caom2 import observation
-from caom2 import ObservationReader
 from datetime import datetime
 
 THIS_DIR = os.path.dirname(os.path.realpath(__file__))
 
 class TestCaom2Integration(unittest.TestCase):
+    
+    logger = logging.getLogger('TestCaom2Integration')
+    
+    def runTest(self):
+        self.test_create_and_visit()
             
     def test_create_and_visit(self):
         
+        print ('-- START: test_create_and_visit --')
```

Comment:
Can you use logging instead of print? That way the output goes to the same location.